docs/nddata/ccddata.rst

```diff
@@ -138,6 +138,21 @@ The ``wcs`` attribute of a `~astropy.nddata.CCDData` object can be set two ways.
 Either way, the ``wcs`` attribute is kept up to date if the
 `~astropy.nddata.CCDData` image is trimmed.
 
+PSF
++++
+
+The ``psf`` attributes of a `~astropy.nddata.CCDData` object can be set two ways.
+
++ If the FITS file has an image HDU extension matching the appropriate name (defaulted to ``"PSFIMAGE"``), the ``psf`` attribute is loaded from that image HDU.
+
++ The PSF can also be provided when the `~astropy.nddata.CCDData` object is
+  constructed with the ``psf`` argument.
+
+The ``psf`` attribute should be a normalized image representing the PSF at the center of the `~astropy.nddata.CCDData`, sized appropriately for the data; users are responsible for managing and interpreting it in context.
+For more on normalizing a PSF image, see :ref:`astropy:kernel_normalization`.
+
+The ``psf`` attribute is set to ``None`` in the output of an arithmetic operation, no matter the inputs. A warning message is emitted if either of the input images contain a non-``None`` psf; users are responsible for determining the appropriate thing to do in that context.
```

Comment:
p.s. Would be nice to break long sentences into shorter lines.

```suggestion
The ``psf`` attribute is set to `None` in the output of an arithmetic operation, no matter the inputs. A warning message is emitted if either of the input images contain a non-`None` PSF; users are responsible for determining the appropriate thing to do in that context.
```

Reply:
I have the opposite opinion on this. Splitting one sentence into multiple lines makes partial changes in future PRs easier to review. Though it is annoying if someone ends up moving words up/down the lines. So, there is no foolproof solution.

I don't think there is an official stance but if you look at the rest of the docs, they are all broken up into shorter lines.